builder-classic-22/builder.toml

```diff
@@ -0,0 +1,111 @@
+description = "Classic (shimmed) buildpack builder for Heroku-22 stack, based on ubuntu:22.04 base image"
+
+[stack]
+id = "heroku-22"
+build-image = "heroku/heroku:22-cnb-build"
+run-image = "heroku/heroku:22-cnb"
+
+[lifecycle]
+version = "0.14.0"
+
+[[buildpacks]]
+  id = "heroku/java"
+  uri = "https://cnb-shim.herokuapp.com/v1/heroku/java?version=0.3.1&name=Java"
```

Comment:
I presume the `version` param here is mostly redundant, since it doesn't affect the returned buildpack other than what's put into `buildpack.toml` - since the latest stable release is always returned?

If so, should we just zero the version out or something, to indicate it's not actually real?

Longer term I guess we could try and make cnb-shim fetch the version from the original buildpack registry, for shims using the alias naming scheme, if we actually cared about the version?

Reply:
According to https://github.com/heroku/pack-images/pull/44, these version numbers are/were meaningful for cache busting. I don't know which cache that this busts, but if that's true, we'd quickly increment past `0.0.0`.

The existing numbers were copied from `heroku/buildpacks:20`.

Reply:
@joshwlewis I can't see any caching in cnb-shim's web server:
https://github.com/heroku/cnb-shim/blob/main/server/main.go

...nor any mention of "cache" in the git history for cnb-shim.

Looking at the pack-images repo state at the time of #44, I see caching in the Circle CI config that may have been the issue here?
https://github.com/heroku/pack-images/blob/1a646e80102df84bd5440d11209cb0c49e9bead0/.circleci/config.yml#L19-L22

> The existing numbers were copied from heroku/buildpacks:20.

Yeah I was just thinking that since this is a brand new image with "parity with Heroku" in mind, we're kind starting with a clean slate :-)

I'm happy either way though.